packages/paste-core/primitives/combobox/stories/mutliselect.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import difference from 'lodash/difference';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Box} from '@twilio-paste/box';
+import {Input} from '@twilio-paste/input';
+import {Label} from '@twilio-paste/label';
+import {FormPill, FormPillGroup, useFormPillState} from '@twilio-paste/form-pill-group';
+import {ComboboxListbox, ComboboxListboxGroup, ComboboxListboxOption} from '@twilio-paste/combobox';
+
+import type {StoryFn} from '@storybook/react/dist/ts3.9/client/preview/types-6-0';
+
+import {ComboboxPrimitive, useComboboxPrimitive, useMultiSelectPrimitive} from '../src';
+
+const items = ['Alert', 'Anchor', 'Button', 'Card', 'Heading', 'List', 'Modal', 'Paragraph'];
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Primitives/Combobox',
+  component: ComboboxPrimitive,
+};
+
+const {FunctionAddSelectedItem, FunctionRemoveSelectedItem} = useMultiSelectPrimitive.stateChangeTypes;
+
+export const BasicMultiCombobox: StoryFn = () => {
+  const seed = useUIDSeed();
+  const [filteredItems, setFilteredItems] = React.useState([...items].sort());
+
+  const formPillState = useFormPillState();
+
+  const {
+    getSelectedItemProps,
+    getDropdownProps,
+    addSelectedItem,
+    removeSelectedItem,
+    selectedItems,
+  } = useMultiSelectPrimitive({
+    onSelectedItemsChange: ({type, selectedItems: currentSelectedItems}) => {
```

Comment:
❓ Why do we need both `onSelectedItemsChange` and `onSelectedItemChange`?

Reply:
the way this works currently it leverages `onSelectedItemChange` from `useCombobox` (which is a callback for the on-click handlers for the options in the dropdown) to trigger the appropriate action from `useMultipleSelection` to either add or remove the selected item to the collection of selected items.

This is a callback that runs when that selection changes. You can think of it as an alternative to adding `useEffect` to handle the side effect inside this method body, which is adjusting the list of filtered items (if a selected item is removed, it adds it back, if an item is selected, it is removed from the filtered items).